pkg/torcx/paths.go

```diff
@@ -26,11 +26,14 @@ const (
 	DefaultBaseDir = "/var/lib/torcx/"
 	// DefaultConfDir is the default torcx config directory
 	DefaultConfDir = "/etc/torcx/"
+
 	// VendorStorePath is the vendor store path
-	VendorStorePath = VENDOR_DIR + "/store/"
+	VendorStorePath = VendorDir + "store/"
+	// VendorProfile is the vendor (lower) profile
+	VendorProfile = VendorDir + VendorProfileName + ".json"
```

Comment:
After a second thought, I think you are completely right and my comment above doesn't really add anything. I'll revert to use what you referenced above, `/usr/share/torcx/profiles/vendor.json`.